src/middlewared/middlewared/plugins/sysctl/sysctl_info.py

```diff
@@ -18,6 +19,11 @@ async def get_value(self, sysctl_name):
             raise CallError(f'Unable to retrieve value of "{sysctl_name}" sysctl : {cp.stderr.decode()}')
         return cp.stdout.decode().split('=')[-1].strip()
 
+    def get_default_arc_max(self):
+        # Default value of arc max on systems
+        # https://github.com/truenas/zfs/blob/f38f3a4a3ee430a15fa221ba9fc2b6b8fd17c040/module/os/linux/zfs/arc_os.c#L85
+        return int(psutil.virtual_memory().total / 2)
```

Comment:
For paranoia about conversions between float and int 😅 
```suggestion
        return psutil.virtual_memory().total // 2
```